assets/scss/_custom.scss

```diff
@@ -1059,10 +1059,11 @@ div.alert > em.javascript-required {
   flex-grow: 1;
   overflow-x: hidden;
   width: auto;
-}
 
-.search-bar:focus-within {
-  border: 2.5px solid rgba(47, 135, 223, 0.7);
+  &:focus-within {
+    outline: 1.5px solid rgba(47, 135, 223, 0.7);
+    border: 1px solid rgba(47, 135, 223, 0.7);
```

Comment:
do we need to set a `border` at all? Maybe `outline` is enough?

Reply:
That's how that would look:

<img width="550" alt="grafik" src="https://github.com/kubernetes/website/assets/32395585/616c2968-8aca-4d65-8ae5-fb3e7f53088c">

<img width="550" alt="grafik" src="https://github.com/kubernetes/website/assets/32395585/4afb3886-f6e5-4618-8b7a-0b491dbedce4">

I don't have any strong feelings, although I personally slightly prefer the extra feedback the additional outline provides:

<img width="555" alt="grafik" src="https://github.com/kubernetes/website/assets/32395585/b6d54135-0ad4-4668-b891-d510476b0c1d">

But I'll leave this up to you to decide. Is there a design system or document within the project that may define this?

Reply:
> Is there a design system or document within the project that may define this?

I believe not. +1 for the border though, I think we should keep it